Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+namespace Gordon360.Extensions.System;
+
+static public class DateTimeExtensions
+{
+    /// <summary>
+    /// Helper method that casts given date time to type UTC without changing the time
+    /// </summary>
+    /// <param name="dateTime"></param>
+    /// <returns>dateTime with type UTC</returns>
+    static public DateTime SpecifyUtc(this DateTime dateTime) => DateTime.SpecifyKind((DateTime)dateTime, DateTimeKind.Utc);
```

Comment:
Since this is the non-nullable version, you shouldn't need to cast the input, right?
```suggestion
    static public DateTime SpecifyUtc(this DateTime dateTime) => DateTime.SpecifyKind(dateTime, DateTimeKind.Utc);
```

Reply:
Yep, didn't catch that update.